test/integration/other/database-indexes.js

```diff
@@ -0,0 +1,73 @@
+const { sql } = require('slonik');
+const { curry, equals, slice } = require('ramda');
+
+const { testContainer } = require('../setup');
+
+const startsWith = curry((prefix, arr) => equals(prefix, slice(0, prefix.length, arr)));
+
+const idxNameFor = fk => `idx_fk_${fk.local_tbl_name}_${fk.local_col_names}`;
+const colsForIdx = fk => fk.local_col_names.map(col => `"${col}"`).join(', ');
+const createIdxStatement = fk => `CREATE INDEX ${idxNameFor(fk)} ON "${fk.local_tbl_name}" (${colsForIdx(fk)});`;
+
+describe('database indexes', () => {
+  it('should define indexes on both sides of foreign key relationships', testContainer(async ({ all }) => {
+    const existingIndexes = await all(sql`
+      SELECT tbl_class.relnamespace::regnamespace::text AS tbl_schema
+           , tbl_class.relname AS tbl_name
+           , idx_class.relname AS idx_name
+           , pg_index.indkey::int2[] AS indexed_columns
+        FROM pg_index
+        JOIN pg_class AS idx_class ON idx_class.oid=pg_index.indexrelid
+        JOIN pg_class AS tbl_class ON tbl_class.oid=pg_index.indrelid
+        WHERE tbl_class.relnamespace::regnamespace::text NOT LIKE 'pg_%'
+    `);
+
+    const foreignKeys = await all(sql`
+    SELECT   local_tbl_class.relnamespace::regnamespace::text AS   local_tbl_schema
+         ,   local_tbl_class.relname                          AS   local_tbl_name
+         , foreign_tbl_class.relnamespace::regnamespace::text AS foreign_tbl_schema
+         , foreign_tbl_class.relname                          AS foreign_tbl_name
```

Comment:
These aren't actually used.  Given how arcane these tables are, and that perf isn't an issue here, it might be helpful to keep them to make the surrounding code more intelligible.